woo-durianpay/woo-durianpay.php

```diff
@@ -737,14 +737,31 @@ private function getOrderCreationData($orderId)
                 'customer'        => $this->getCustomerInfo($order),
                 'order_ref_id'    => strval($orderId),
                 'items'           => $this->getCartInfo(),
-                'expiry_date'     => $this->getSetting('expired_order'),
+                'expiry_date'     => $this->getExpiryTimestamp($this->getSetting('expired_order')),
                 'shipping_fee'    => $this->getShippingFee($order),
                 'is_payment_link' => $this->getPaymentNotification()
             );
 
             return $data;
         }
 
+        private function getExpiryTimestamp($expiryRequest) {
+            # expiry request is in the format hours:minutes:seconds
+            # hours can have a value of more than 24, meaning you can add 1 or more days for an expiry date
+            # minutes can have a value of more than 60, meaning you can add 1 or more hours for an expiry date
+            # likewise for seconds, a value of more than 60 means we are adding 1 or more minutes for an expiry date
+            # example => 100:100:100 100 hours, 100 minutes and 100 seconds meaning order will expire 4 days, 5 hours, 41 minutes and 40 seconds from now
```

Comment:
Any input field from woocommerce admin page? If so please add the hint or description on how to use this field @PLSV7Durianpay 

cc @akhamat-durianmoney

Reply:
I have tested this with @arifbudurian and I have ensured that the order expiry methods work as planned